digits/model/tasks/torch_train.py

```diff
@@ -133,14 +135,12 @@ def task_arguments(self, resources, env):
                 '--save=%s' % self.job_dir,
                 '--snapshotPrefix=%s' % self.snapshot_prefix,
                 '--snapshotInterval=%s' % self.snapshot_interval,
+                '--batchSize=%d' % self.batch_size,
```

Comment:
this has also been removed on master, you need to keep the code from https://github.com/NVIDIA/DIGITS/blob/master/digits/model/tasks/torch_train.py#L141-L142